<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StandardDialogs</class>
 <widget class="QMainWindow" name="StandardDialogs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StandardDialogs</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>3</y>
      <width>279</width>
      <height>165</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btColor">
       <property name="text">
        <string>Select Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btFile">
       <property name="text">
        <string>Select File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btMessage">
       <property name="text">
        <string>Show Error Message</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btInput">
       <property name="text">
        <string>Input Text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Push a button...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>285</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
